### DISCUSSION TITLE
Quasar v1 support - time to upgrade to v2

### DISCUSSION THREAD

**Discussion by rstoenescu** [quasarframework/quasar#15229] [Announcements]
Dear Quasar Dev Community,

We are officially sunsetting version 1 of Quasar as of the **30th of June** this year (2023). At that point, Quasar v2 will have been generally available and production ready for 2 years. We need to save the effort of upkeeping v1 for bigger and better things to come.

If you haven't already [migrated to Quasar v2](https://quasar.dev/start/upgrade-guide#migrate-to-v2-from-v1), we highly recommend doing so, soon.

If you have any more questions, please feel free to comment below.

Your Quasar Framework Team!

**Comment by dev-oiram** [comment]
@rstoenescu is there some docs to help with this migration for current projects created with version 1?

**Reply by rstoenescu** [reply]
Yes, of course. https://quasar.dev/start/upgrade-guide

**Reply by dev-oiram** [reply]
> Yes, of course. https://quasar.dev/start/upgrade-guide

Thanks for the link

**Comment by dev-oiram** [comment]
@rstoenescu does this mean that projects created with version 1 will no longer work after the 30th of June?
- Like running `npm ci` and not able to find quasar v1 dependencies

**Reply by rstoenescu** [reply]
No. It just means that Quasar v1 (and its CLI) will no longer receive updates (bugfixing, new features, etc). You will still be able to use your Quasar v1 projects as these dependencies will not (and also: cannot) be "unpublished" anyway.
